### PR TITLE
gxm: Improve cube textures

### PR DESCRIPTION
**Pull request by scribam**
Testing required

### CODE REVIEW

File: vita3k/modules/SceGxm/SceGxm.cpp [hunk 1]
```diff
@@ -2617,11 +2617,14 @@ EXPORT(int, sceGxmTextureInitCube, SceGxmTexture *texture, Ptr<const void> data,
...
 }
 
-EXPORT(int, sceGxmTextureInitCubeArbitrary, SceGxmTexture *texture, const void *data, SceGxmTextureFormat texFormat, uint32_t width, uint32_t height, uint32_t mipCount) {
+EXPORT(int, sceGxmTextureInitCubeArbitrary, SceGxmTexture *texture, Ptr<const void> data, SceGxmTextureFormat texFormat, uint32_t width, uint32_t height, uint32_t mipCount) {
     if (!texture) {
         return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
     }
-    return UNIMPLEMENTED();
+
+    const int result = init_texture_base(export_name, texture, data, texFormat, width, height, mipCount, SCE_GXM_TEXTURE_CUBE_ARBITRARY);
```
Comment (Zangetsu38) [inline vita3k/modules/SceGxm/SceGxm.cpp:2625]: Is same init cube, is keep stub, cube request also works for shader

**Review by pent0 — COMMENTED** [review]
Cube can't be uploaded the same as normal texture for now (it's store a bit different in memory), but if does not crash and increase compatibility, it's ok

File: vita3k/modules/SceGxm/SceGxm.cpp [hunk 2]
```diff
@@ -2617,11 +2617,14 @@ EXPORT(int, sceGxmTextureInitCube, SceGxmTexture *texture, Ptr<const void> data,
     return result;
 }
 
-EXPORT(int, sceGxmTextureInitCubeArbitrary, SceGxmTexture *texture, const void *data, SceGxmTextureFormat texFormat, uint32_t width, uint32_t height, uint32_t mipCount) {
+EXPORT(int, sceGxmTextureInitCubeArbitrary, SceGxmTexture *texture, Ptr<const void> data, SceGxmTextureFormat texFormat, uint32_t width, uint32_t height, uint32_t mipCount) {
     if (!texture) {
         return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
     }
-    return UNIMPLEMENTED();
+
```
Comment (Zangetsu38) [inline vita3k/modules/SceGxm/SceGxm.cpp:2624]:
```suggestion
    STUBBED("Stub InitCubeArbitrary");
```

**Review by Zangetsu38 — CHANGES_REQUESTED** [review]
Only need add this, is like initcube, request works on shader

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**